ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -44,3 +44,43 @@ nginx_sites:
       balancer: "{{ deploy_env }}_formplayer/"
       proxy_next_upstream_tries: 1
       proxy_read_timeout: 900s
+    - name: /files
```

Comment:
it looks like based on our current templates, we dont have a way to conditionally add locations. I can try to change our template to allow that, something like `env_locations` that pulls from the vars file, and could be added to the locations list here. does that seem worth it?